src/Inventory/CreateList.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import {
+	StyleSheet,
+	Image,
+	View,
+	TouchableHighlight,
+	FlatList,
+	Text,
+	TextInput,
+	Linking
+} from 'react-native';
+import { colors } from '../../config/styles';
+import { List, ListItem, Card, Button, Icon, FormLabel, FormInput, FormValidationMessage } from 'react-native-elements';
+import { SafeAreaView } from 'react-navigation';
+import Header from 'react-navigation/src/views/Header/Header';
+import Meteor, { createContainer } from 'react-native-meteor';
+import settings from '../../config/settings';
+
+class CreateList extends React.Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			name: ''
+		};
+	}
+
+	_createNewList() {
+		if (this.state.name.length === 0) {
+			console.log('name cannot be empty'); // eslint-disable-line
+			return;
+		}
+
+		Meteor.call('inventorylists.create', this.state.name);
+		
+		this.setState({ name: '' });
+		this.props.navigation.goBack();
+	}
+
+	render() {
+		return (
+			<SafeAreaView style={{ flex: 1 }}>
+				<Header scene={{index: 0}}
```

Comment:
It's an issue with a modal since it won't have a header. I followed [this article](https://medium.com/4thought-studios/better-modals-in-react-navigation-56c649e0249) to make it work the way it does. We might be able to figure out a different workaround or even change the style altogether.